lib/pdf_print_configuration.dart

```diff
@@ -18,6 +21,7 @@ class PrintPdfConfiguration {
     required this.targetName,
     this.printSize = PrintSize.A4,
     this.printOrientation = PrintOrientation.Portrait,
+    this.linksClickable = false,
```

Comment:
@penkzhou here it is as default value. 
Default value is false and on that way default printing is the same as current implementation. It is not set as true by default because in UIMarkupTextPrintFormatter, images must be set as data base64.